### PR TITLE
Update README with more info

### PR DESCRIPTION
**Pull request by cbrevik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,33 @@
-# [WIP] Spellchecker Bot
+# Probot Spellchecker
```
[the remark this answers is not in the document]
Comment (cbrevik) [reply README.md:1]: Ooops, forgot to check for lower-case ignored words 😅

File: README.md [hunk 2]
```diff
@@ -1,9 +1,33 @@
...
+
+A simple probot which will spellcheck your PRs. Currently supporting just `.md` files. There's no reason it can't support more filestypes, contributions are welcome!
```
[the remark this answers is not in the document]
Comment (mikaelbr) [reply README.md]: Was that intentional or did it just actually prove it's worth?

[the remark this answers is not in the document]
Comment (cbrevik) [reply README.md]: That was not intentional, so that is pretty funny!

[the remark this answers is not in the document]
Comment (mikaelbr) [reply README.md]: That is soo good! 💃

**Pull request merged**